dluhc-component-library/src/components/timetableForm/TimetableForm.tsx

```diff
@@ -0,0 +1,58 @@
+import { useMemo, useState } from "preact/hooks";
+import DescriptionPage from "./components/DescriptionPage";
+import PublishedDatePage from "./components/PublishedDatePage";
+import TitlePage from "./components/TitlePage";
+
+const TIMETABLE_FORM_PAGES = new Map([
```

Comment:
I didnt think about actually using a `Map()` i was thinking an object, however i think Map keeps the ordering though so we may not need the `TIMETABLE_PAGE_IDS` variable. Worth looking into

Reply:
As discussed offline, the components for stages will need props to be passed in. My current implementation is to have a list of objects where the object has the id and the component. Once I add stages I'll put the props on those objects.